sematic/db/models/factories.py

```diff
@@ -149,20 +151,36 @@ def make_func_path(future: AbstractFuture) -> str:
     return f"{future.calculator.__module__}.{future.calculator.__name__}"
 
 
-def make_artifact(value: Any, type_: Any) -> Tuple[Artifact, bytes]:
+class StorageNamespace(enum.Enum):
+    artifacts = "artifacts"
+    futures = "futures"
```

Comment:
This isn't used. Shouldn't this new functionality be propagated to the usage of `store_future_bytes` in `worker.py`?

Reply:
Done.